components/runs.html

```diff
@@ -41,7 +41,7 @@
         async connectedCallback () {
           super.connectedCallback()
 
-          if (!this.testRuns) {
+          if (this.testRunResources) {
```

Comment:
I'm trying to understand this change—I don't see any other changes that involve `this.testRunResources`. Any details would be appreciated 👍

Reply:
TestRun is an array of the JSON objects, which can be passed in directly. TestRunResources are URLs to fetch the JSON objects. This PR uses both (the TestRun for the diff is passed as a JSON object).

This will probably change anyway, since the single-result view 'diff' isn't needed, so will probably keep 'diff' as a separate property on the Polymer element.

Reply:
Got it, thanks for the details!